pkg/postgres/gorm/largeobject/large_objects.go

```diff
@@ -0,0 +1,169 @@
+package largeobject
+
+import (
+	"errors"
+	"io"
+
+	"gorm.io/gorm"
+)
+
+// LargeObjects is used to access the large objects API with gorm CRM.
+//
+// This is originally created with similar API with existing github.com/jackc/pgx
+// For more details see: http://www.postgresql.org/docs/current/static/largeobjects.html
+type LargeObjects struct {
+	*gorm.DB
```

Comment:
should this be gorm.Tx?

Reply:
No. gorm.Tx is only an interface to have access to transaction and context operation. That is not enough for large object operations.